bukuserver/templates/bukuserver/bookmark_details.html

```diff
@@ -4,6 +4,7 @@
 {% block tail %}
   {{ super() }}
   {{ buku.details_formatting() }}
+  <script>$('#fa_filter, .table.searchable a').attr('tabindex', 1)</script>
```

Comment:
I wasn't sure about what UX would be better for the details page, so instead of focusing on the filter immediately I made both the filter and the link inside the table to be the first elements that the Tab key would focus on.